barretenberg/cpp/pil/avm/constants_gen.pil

```diff
@@ -4,8 +4,8 @@ namespace constants(256);
     pol MAX_NULLIFIERS_PER_CALL = 16;
     pol MAX_PUBLIC_CALL_STACK_LENGTH_PER_CALL = 16;
     pol MAX_L2_TO_L1_MSGS_PER_CALL = 2;
```

Comment:
I had to bump these constants to be able to pass 2 hiding points and 2 logs via "transient storage" in private refunds flow.